### PR TITLE
Updateservice responses

### PR DESCRIPTION
**Pull request by sunithasomanna**
Updated responses for Updateservice
No change for Fabrics as discussed in today's standup

### CODE REVIEW

**Review by jeevan-kamkar — APPROVED** [review]

**Review by chauberahul1993 — APPROVED** [review]

File: docs/README.md [hunk 1]
```diff
@@ -8391,14 +8396,13 @@ curl -i GET \
...
-   ​   "Description":"SoftwareInventory view",
-   ​   "Name":"SoftwareInventory",
-   ​   "Members":null,
-   ​   "Members@odata.count":0​
+{  "@odata.context":"/redfish/v1/$metadata#SoftwareInventoryCollection.SoftwareCollection",
+   "@odata.id":"/redfish/v1/UpdateService/SoftwareInventory",
+   "@odata.type":"#SoftwareInventoryCollection.SoftwareInventoryCollection",
+   "Description":"SoftwareInventory view",
+   "Name":"SoftwareInventory",
+   "Members":null,
```
Comment (amar-shalgar) [inline docs/README.md]: as per redfish members cannot be null, if no value it should be empty array

File: docs/README.md [hunk 2]
```diff
@@ -8474,11 +8478,8 @@ curl -i POST \
    -d \
 '{
 "ImageURI": "<URI_of_the_firmware_image>",
-"Password": "{password}",
 "Targets": ["/redfish/v1/Systems/{ComputerSystemId}"],
 "@Redfish.OperationApplyTime": "OnStartUpdateRequest"
```
Comment (amar-shalgar) [inline docs/README.md:8482]: Change @Redfish.OperationApplyTimeSupport in the request parameters section to valid property

**Review by amar-shalgar — CHANGES_REQUESTED** [review]

File: docs/README.md (hunk 1 above)
Comment (sunithasomanna) [reply docs/README.md]: fixed

File: docs/README.md (hunk 2 above)
Comment (sunithasomanna) [reply docs/README.md:8482]: fixed

**Review by amar-shalgar — APPROVED** [review]

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**